app/helpers/collection_time_helper.rb

```diff
@@ -13,4 +13,16 @@ def collection_start_date(date)
   def current_collection_start_date
     Time.zone.local(current_collection_start_year, 4, 1)
   end
+
+  def current_collection_end_date
+    Time.zone.local((current_collection_start_year + 1), 3, 31)
+  end
+
+  def next_collection_start_year
+    current_collection_start_year + 1
+  end
+
+  def previous_collection_start_year
+    current_collection_start_year - 1
+  end
```

Comment:
Might be worth a rebase as I'd already implemented `current_collection_end_date` in a commit merged earlier this week.